Automattic/jetpack
# JITM: Add external link icon to CTAs that opens new tab.

Certain JITM CTAs open new tabs, those that do, should have an external link icon. This PR updates the JITM to render external link icon.

**Before**:
<img width="640" alt="Screen Shot 2023-03-23 at 8 17 04 PM" src="https://user-images.githubusercontent.com/56598660/227201294-1e8df1c0-4738-4a3e-9ae7-d84f264813ed.png">

**After**:
<img width="1096" alt="Screen Shot 2023-03-23 at 7 43 54 PM" src="https://user-images.githubusercontent.com/56598660/227201180-f7414e04-ed58-4156-b952-2b654849a941.png">




Fixes #29552

## Proposed changes:
* Update JITM package to render external link icons when CTA opens a new tab.

### Other information:


## Jetpack product discussion
A P2 post discussing about how to improve the current JITM should be found in the link below:
p1HpG7-l9I-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:






* Spin up a JN site and make sure you are running it with Jetpack Beta pointing to this branch.
* Connect your JN site to your WP account
* Once connected, go to JP dashboard screen (/wp-admin/admin.php?page=jetpack#/dashboard)
* Confirm JITM with 'Sign up' CTAs renders with external link icon. clicking this CTA should open a new tab.
<img width="1064" alt="Screen Shot 2023-03-23 at 9 03 35 PM" src="https://user-images.githubusercontent.com/56598660/227213073-6d5daefc-095c-4244-a9f3-6342c948e24a.png">
* Try to activate more Jetpack modules in My Jetpack and confirm that some of the CTAs also work as expected.
<img width="1108" alt="Screen Shot 2023-03-23 at 9 11 49 PM" src="https://user-images.githubusercontent.com/56598660/227214821-19ef217d-8542-44ab-9ec7-85cb0890d4fa.png">
* There are a few JITM with CTAs that do not open a new tab. The external link icon should not be rendered in this case. eg
<img width="1108" alt="Screen Shot 2023-03-23 at 9 11 49 PM" src="https://user-images.githubusercontent.com/56598660/227215343-c8bbd4b5-d290-4384-b887-bf0a628f0796.png">
